src/main/java/com/org/jenkins/custom/jenkins/distribution/service/PackagerController.java

```diff
@@ -34,4 +42,32 @@
             return new ResponseEntity(HttpStatus.NOT_FOUND);
         }
     }
+
+    /**
+     * @param postPayload Cofiguration on the front end that gets posted as a string
+     * @return Packager-config.yml
+     */
+    @PostMapping (path = "/downloadPackageConfiguration")
+    public ResponseEntity<?> downloadPackageConfig(@RequestBody String postPayload) {
+        LOGGER.info("Request Received for downloading configuration with params" + postPayload);
```

Comment:
`postPayload` might be a large text. It's not a good practice to print it especially as `info` level.

Reply:
Yes thanks :)